charmcraft/models/charmcraft.py

```diff
@@ -74,7 +74,23 @@ class BasesConfiguration(
     alias_generator=lambda s: s.replace("_", "-"),
     frozen=True,
 ):
-    """Definition of build-on/run-on combinations."""
+    """Definition of build-on/run-on combinations.
```

Comment:
This is a shortened version of https://juju.is/docs/sdk/charmcraft-yaml#heading--bases